### PR TITLE
test: update OpenProperty with SelectEntityByName

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
- This exercise intents to replace OpenPropertyPane() js helper method with ts helper SelectEntityByName() and other methods on use case basis

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ClientSide/Autocomplete/autocomplete_spec.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   before(() => {
     cy.addDsl(dsl);
   });
   it("1. Opens autocomplete for bindings", () => {
-    cy.selectEntityByName("TestModal");
-    cy.wait(3000);
-    cy.selectEntityByName("Aditya");
-    cy.openPropertyPane("buttonwidget");
+    _.entityExplorer.ExpandCollapseEntity("TestModal");
+    _.entityExplorer.SelectEntityByName("Aditya");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Autocomplete/autocomplete_spec.js:11]: SelectEntityByName takes a 2nd parameter which it will expand. Use that here.

File: app/client/cypress/e2e/Regression/ClientSide/Binding/ChartText_spec.js [hunk 2]
```diff
@@ -2,21 +2,24 @@ const commonlocators = require("../../../../locators/commonlocators.json");
...
+import * as _ from "../../../../support/Objects/ObjectsCore";
 
 describe("Text-Chart Binding Functionality", function () {
   before(() => {
     cy.addDsl(dsl);
   });
 
   it("1. Text-Chart Binding Functionality View", function () {
-    cy.openPropertyPane("textwidget");
+    _.entityExplorer.ExpandCollapseEntity("Container3");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/ChartText_spec.js:13]: Same as above. Correct this in all specs & re-send for review.

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TableWidgets_NavigateTo_Validation_spec.js [hunk 3]
```diff
@@ -4,11 +4,7 @@ const testdata = require("../../../../fixtures/testdata.json");
 const dsl = require("../../../../fixtures/tableWidgetDsl.json");
 const dsl2 = require("../../../../fixtures/displayWidgetDsl.json");
 const pageid = "MyPage";
-import { ObjectsRegistry } from "../../../../support/Objects/Registry";
-import * as _ from "../../../../support/Objects/ObjectsCore";
-const agHelper = ObjectsRegistry.AggregateHelper;
-const propPane = ObjectsRegistry.PropertyPane;
-let entityExplorer = ObjectsRegistry.EntityExplorer;
+import {agHelper,propPane,entityExplorer} from "../../../../support/Objects/ObjectsCore"
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TableWidgets_NavigateTo_Validation_spec.js:7]: Yes this is the approach forward!

Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/Binding/TableWidgets_NavigateTo_Validation_spec.js]: sure

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2_Api_spec.js [hunk 4]
```diff
@@ -26,10 +26,10 @@ describe("Test Create Api and Bind to Table widget V2", function () {
...
 
   it("2. Test_Validate the Api data is updated on Table widget", function () {
-    _.entityExplorer.ExpandCollapseEntity("Widgets");
-    _.entityExplorer.ExpandCollapseEntity("Container3");
-    _.entityExplorer.SelectEntityByName("Table1");
-    cy.openPropertyPane("tablewidgetv2");
+    entityExplorer.ExpandCollapseEntity("Widgets");
+    entityExplorer.ExpandCollapseEntity("Container3");
+    entityExplorer.SelectEntityByName("Table1");
+    entityExplorer.SelectEntityByName("Table1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2_Api_spec.js:32]: Why twice?

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2_Api_spec.js [hunk 5]
```diff
@@ -57,11 +57,11 @@ describe("Test Create Api and Bind to Table widget V2", function () {
...
   it("3. Validate onSearchTextChanged function is called when configured for search text", function () {
-    _.entityExplorer.ExpandCollapseEntity("Widgets");
-    _.entityExplorer.ExpandCollapseEntity("Container3");
-    _.entityExplorer.SelectEntityByName("Table1");
+    entityExplorer.ExpandCollapseEntity("Widgets");
+    entityExplorer.ExpandCollapseEntity("Container3");
+    entityExplorer.SelectEntityByName("Table1");
 
-    cy.openPropertyPane("tablewidgetv2");
+    entityExplorer.SelectEntityByName("Table1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2_Api_spec.js:64]: Why twice?

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TableWidgets_NavigateTo_Validation_spec.js [hunk 6]
```diff
@@ -27,13 +29,11 @@ describe("Table Widget and Navigate to functionality validation", function () {
     cy.addDsl(dsl2);
     // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.wait(500);
-    cy.CheckAndUnfoldEntityItem("Pages");
+    entityExplorer.ExpandCollapseEntity("Pages");
     cy.get(`.t--entity-name:contains("${pageid}")`).should("be.visible");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TableWidgets_NavigateTo_Validation_spec.js:33]: Not needed.

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2_Api_spec.js [hunk 7]
```diff
@@ -57,11 +57,11 @@ describe("Test Create Api and Bind to Table widget V2", function () {
   });
 
   it("3. Validate onSearchTextChanged function is called when configured for search text", function () {
-    _.entityExplorer.ExpandCollapseEntity("Widgets");
-    _.entityExplorer.ExpandCollapseEntity("Container3");
-    _.entityExplorer.SelectEntityByName("Table1");
+    entityExplorer.ExpandCollapseEntity("Widgets");
+    entityExplorer.ExpandCollapseEntity("Container3");
+    entityExplorer.SelectEntityByName("Table1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2_Api_spec.js:62]: Can be combined into single `SelectEntityByName`

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2_Api_spec.js [hunk 8]
```diff
@@ -26,10 +26,10 @@ describe("Test Create Api and Bind to Table widget V2", function () {
   });
 
   it("2. Test_Validate the Api data is updated on Table widget", function () {
-    _.entityExplorer.ExpandCollapseEntity("Widgets");
-    _.entityExplorer.ExpandCollapseEntity("Container3");
-    _.entityExplorer.SelectEntityByName("Table1");
-    cy.openPropertyPane("tablewidgetv2");
+    entityExplorer.ExpandCollapseEntity("Widgets");
+    entityExplorer.ExpandCollapseEntity("Container3");
+    entityExplorer.SelectEntityByName("Table1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2_Api_spec.js:31]: Can be combined into single `SelectEntityByName`

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/DynamicHeight_Visibility_spec.js [hunk 9]
```diff
@@ -8,11 +9,11 @@ describe("Dynamic Height Width validation for Visibility", function () {
...
     //changing the Text Name and verifying
     cy.wait(3000);
-    cy.openPropertyPane("containerwidget");
+    entityExplorer.SelectEntityByName("Container1", "Widgets");
     cy.changeLayoutHeightWithoutWait(commonlocators.autoHeight);
-    cy.openPropertyPaneWithIndex("inputwidgetv2", 0);
+    entityExplorer.SelectEntityByName("Input1", "Container1");
     cy.changeLayoutHeightWithoutWait(commonlocators.autoHeight);
-    cy.openPropertyPaneWithIndex("inputwidgetv2", 1);
+    entityExplorer.SelectEntityByName("Input2", "Container1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/DynamicHeight_Visibility_spec.js:16]: Since it already in expanded state from from , `Container1` is not needed here.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js [hunk 10]
```diff
@@ -14,12 +11,12 @@ describe("Dynamic Height Width validation list widget", function () {
     cy.wait(3000); //for dsl to settle
     cy.get(explorer.addWidget).click();
     cy.dragAndDropToCanvas("multiselecttreewidget", { x: 300, y: 500 });
-    cy.openPropertyPane("listwidget");
+    entityExplorer.SelectEntityByName("List1", "Widgets");
     //Widgets which were not possible to be added to list widget cannot be pasted/moved into the list widget with multitreeselect
-    cy.openPropertyPane("multiselecttreewidget");
+    entityExplorer.SelectEntityByName("MultiTreeSelect1", "Widgets");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:16]: `Widgets` not needed here, since already expanded after above stmt.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js [hunk 11]
```diff
@@ -14,12 +11,12 @@ describe("Dynamic Height Width validation list widget", function () {
...
     cy.get("body").type(`{${modifierKey}}c`);
     cy.get(".Toastify__toast-body span").should("not.exist");
-    cy.selectEntityByName("List1");
+    entityExplorer.SelectEntityByName("List1", "Widgets");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:19]: `Widgets` not needed here, since already expanded after above stmt.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js [hunk 12]
```diff
@@ -51,14 +51,15 @@ describe("Dynamic Height Width validation list widget", function () {
...
+
         cy.get(commonlocators.generalSectionHeight).should("not.exist");
         //Widgets when moved into the list widget have no dynamic height
-        cy.selectEntityByName("Text3");
+        entityExplorer.SelectEntityByName("Text3", "Widgets");
         cy.moveToStyleTab();
 
         cy.get("body").type(`{${modifierKey}}c`);
-        cy.selectEntityByName("List1");
+        entityExplorer.SelectEntityByName("List1", "Widgets");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:62]: `Widgets` not needed here, since already expanded after above stmt.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js [hunk 13]
```diff
@@ -69,7 +70,10 @@ describe("Dynamic Height Width validation list widget", function () {
         );
         cy.wait(2000);
         entityExplorer.NavigateToSwitcher("Explorer");
-        cy.selectEntityByName("Text3Copy");
+        entityExplorer.SelectEntityByName("List1", "Widgets");
+        entityExplorer.SelectEntityByName("Container1", "List1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:74]: Instead call `ExpandCollapseEntity` here.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js [hunk 14]
```diff
@@ -106,10 +110,10 @@ describe("Dynamic Height Width validation list widget", function () {
                 expect(height).to.not.equal(newheight);
               });
           });
-        cy.selectEntityByName("Text3CopyCopy");
+        entityExplorer.SelectEntityByName("Text3CopyCopy", "Widgets");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:113]: `Widgets` not needed.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js [hunk 15]
```diff
@@ -119,17 +123,15 @@ describe("Dynamic Height Width validation list widget", function () {
...
-        cy.selectEntityByName("Text3CopyCopyCopy");
+        entityExplorer.SelectEntityByName("Text3CopyCopyCopy", "Container1");
         cy.wait(2000);
         cy.get(commonlocators.generalSectionHeight).should("not.exist");
-        cy.selectEntityByName("Text3CopyCopy");
+        entityExplorer.SelectEntityByName("Text3CopyCopy", "Widgets");
         cy.wait(2000);
         cy.get("body").type(`{${modifierKey}}x`);
-        cy.selectEntityByName("List1");
+        entityExplorer.SelectEntityByName("List1", "Widgets");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:134]: `Widgets` not needed.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js [hunk 16]
```diff
@@ -139,7 +141,8 @@ describe("Dynamic Height Width validation list widget", function () {
           200,
         );
         cy.wait(2000);
-        cy.selectEntityByName("Text3CopyCopy");
+        entityExplorer.SelectEntityByName("Container1", "List1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:144]: Why is this needed? if below line is         `entityExplorer.SelectEntityByName("Text3CopyCopy", "Widgets");`

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_spec.js [hunk 17]
```diff
@@ -1,20 +1,21 @@
...
       .invoke("css", "height")
       .then((lheight) => {
         cy.get(commonlocators.generalSectionHeight).should("not.exist");
-        cy.openPropertyPaneWithIndex("textwidget", 0);
+        entityExplorer.SelectEntityByName("Container1", "List1");
+        entityExplorer.SelectEntityByName("Text1", "Container1");
         cy.get(commonlocators.generalSectionHeight).should("not.exist");
         cy.testCodeMirror(textMsg);
-        cy.openPropertyPaneWithIndex("textwidget", 1);
+        entityExplorer.SelectEntityByName("Text2", "Container1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_spec.js:18]: `Container1` not needed.

File: app/client/cypress/e2e/Regression/ServerSide/ApiTests/API_Mustache_spec.js [hunk 18]
```diff
@@ -3,13 +3,15 @@ const commonlocators = require("../../../../locators/commonlocators.json");
...
 const testdata = require("../../../../fixtures/testdata.json");
+import { entityExplorer } from "../../../../support/Objects/ObjectsCore";
 
 describe("Moustache test Functionality", function () {
   beforeEach(() => {
     cy.addDsl(dsl);
   });
   it("1. Moustache test Functionality", function () {
-    cy.openPropertyPane("textwidget");
+    entityExplorer.SelectEntityByName("Aditya", "Widgets");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ServerSide/ApiTests/API_Mustache_spec.js:13]: Use `ExpandCollapseEntity`

Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ServerSide/ApiTests/API_Mustache_spec.js:13]: What cases should we use `ExpandCollapseEntity` here in this case they have renamed a widget as Aditya which is under Widgets.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.ts [hunk 19]
```diff
@@ -0,0 +1,15 @@
+import {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.ts]: What is this spec? Remove it from this PR - to stick to the PR's purpose.

Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.ts]: this was removed 2 days back as part of this commit. https://github.com/appsmithorg/appsmith/pull/24193/commits/99d0b534ba312f8282a8d1abe828c62e683fee9e

Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.ts]: Why is this needed here? - I dint get!

Comment (NandanAnantharamu) [reply app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.ts]: its removed is what i meant, my bad for not being clear

File: app/client/cypress/e2e/Regression/ServerSide/ApiTests/API_Mustache_spec.js [hunk 20]
```diff
@@ -3,13 +3,15 @@ const commonlocators = require("../../../../locators/commonlocators.json");
...
+import { entityExplorer } from "../../../../support/Objects/ObjectsCore";
 
 describe("Moustache test Functionality", function () {
   beforeEach(() => {
     cy.addDsl(dsl);
   });
   it("1. Moustache test Functionality", function () {
-    cy.openPropertyPane("textwidget");
+    entityExplorer.ExpandCollapseEntity("Aditya");
+    entityExplorer.SelectEntityByName("TestTextBox", "Aditya");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ServerSide/ApiTests/API_Mustache_spec.js:14]: Ok, in this case `entityExplorer.ExpandCollapseEntity("Aditya");` was not required. 
Is this what you asked in another comment? Fix this in next PR.

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.js [hunk 21]
```diff
@@ -1,17 +1,17 @@
...
       "Dynamic panel validation for text widget wrt heightDynamic panel validation for text widget wrt heightDynamic panel validation for text widget wrt height Dynamic panel validation for text widget Dynamic panel validation for text widget Dynamic panel validation for text widget";
     cy.addDsl(dsl);
-    cy.wait(3000); //for dsl to settle
-    cy.selectEntityByName("Modal1");
+    entityExplorer.SelectEntityByName("Modal1", "Widgets");
     cy.get(".t--modal-widget")
       .invoke("css", "height")
       .then((mheight) => {
-        cy.openPropertyPane("textwidget");
+        entityExplorer.SelectEntityByName("Text1", "Modal1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.js:14]: Is this needed, when we already have it selected in line # 10?

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_spec.js [hunk 22]
```diff
@@ -1,20 +1,21 @@
...
-    cy.wait(3000); //for dsl to settle
-    cy.openPropertyPane("listwidget");
+    entityExplorer.SelectEntityByName("List1");
     cy.get(".t--widget-listwidget")
       .invoke("css", "height")
       .then((lheight) => {
         cy.get(commonlocators.generalSectionHeight).should("not.exist");
-        cy.openPropertyPaneWithIndex("textwidget", 0);
+        entityExplorer.SelectEntityByName("Container1", "List1");
+        entityExplorer.SelectEntityByName("Text1", "Container1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_spec.js:15]: This can be :
  ```
  entityExplorer.ExpandCollapseEntity("List1");
    entityExplorer.SelectEntityByName("Text1", "Container1");
```

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js (hunk 14 above)
Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:113]: ?

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js (hunk 13 above)
Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:74]: ?

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js (hunk 12 above)
Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/List_TextWidget_Spec.js:62]: ?

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.ts (hunk 19 above)
Comment (Aishwarya-U-R) [reply app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Modal_Widget_spec.ts]: Got it!

File: app/client/cypress/e2e/Regression/ClientSide/Autocomplete/autocomplete_spec.js [hunk 23]
```diff
@@ -1,15 +1,14 @@
...
 describe("Dynamic input autocomplete", () => {
   before(() => {
     cy.addDsl(dsl);
   });
   it("1. Opens autocomplete for bindings", () => {
-    cy.selectEntityByName("TestModal");
-    cy.wait(3000);
-    cy.selectEntityByName("Aditya");
-    cy.openPropertyPane("buttonwidget");
+    entityExplorer.SelectEntityByName("Aditya");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Autocomplete/autocomplete_spec.js:10]: Why is this `SelectEntityByName("Aditya"); `needed?

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TableTextPagination_spec.js [hunk 24]
```diff
@@ -17,17 +21,16 @@ describe("Test Create Api and Bind to Table widget", function () {
...
     /**Bind Api1 with Table widget */
     cy.testJsontext("tabledata", "{{Api1.data}}");
     cy.CheckWidgetProperties(commonlocators.serverSidePaginationCheckbox);
     /**Bind Table with Textwidget with selected row */
-    _.entityExplorer.SelectEntityByName("Text1");
-
+    entityExplorer.SelectEntityByName("Text1", "Widgets");
     cy.testJsontext("text", "{{Table1.selectedRow.avatar}}");
-    _.entityExplorer.SelectEntityByName("Table1");
+    entityExplorer.SelectEntityByName("Table1", "Widgets");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TableTextPagination_spec.js:33]: `Widgets` not needed. Since already expanded via line # 24

File: app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2Widgets_NavigateTo_Validation_spec.js [hunk 25]
```diff
@@ -28,13 +29,11 @@ describe("Table Widget V2 and Navigate to functionality validation", function ()
...
     cy.wait(500);
-    cy.CheckAndUnfoldEntityItem("Pages");
+    entityExplorer.ExpandCollapseEntity("Pages");
     cy.get(`.t--entity-name:contains("${pageid}")`).should("be.visible");
     //Table Widget V2 Functionality with multiple page
-    cy.get(`.t--entity-name:contains("Page1")`)
-      .should("be.visible")
-      .click({ force: true });
-    cy.openPropertyPane("tablewidgetv2");
+    entityExplorer.SelectEntityByName("Page1", "Pages");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/Binding/TableV2Widgets_NavigateTo_Validation_spec.js:35]: is this `entityExplorer.ExpandCollapseEntity("Pages");` needed?

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Container_collapse_undo_redoSpec.js [hunk 26]
```diff
@@ -1,15 +1,16 @@
...
     const modifierKey = Cypress.platform === "darwin" ? "meta" : "ctrl";
     cy.addDsl(dsl);
     cy.wait(3000); //for dsl to settle
-    cy.openPropertyPane("containerwidget");
+    entityExplorer.SelectEntityByName("Container1");
     cy.get(".t--widget-containerwidget")
       .invoke("css", "height")
       .then((height) => {
-        cy.openPropertyPane("buttonwidget");
+        entityExplorer.SelectEntityByName("Button1", "Container1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Container_collapse_undo_redoSpec.js:13]: `entityExplorer.SelectEntityByName("Container1");` is needed?

File: app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Container_Scroll_spec.js [hunk 27]
```diff
@@ -1,12 +1,13 @@
...
 
 describe("Dynamic Height Width validation", function () {
   it("1. Validate change with auto height width for widgets", function () {
     cy.addDsl(dsl);
     cy.wait(3000); //for dsl to settle
-    cy.openPropertyPane("containerwidget");
+    entityExplorer.SelectEntityByName("Container1");
     cy.get(".t--widget-textwidget").trigger("mouseover", { force: true }); // Scroll 'sidebar' to its bottom
-    cy.openPropertyPane("textwidget");
+    entityExplorer.SelectEntityByName("Text1", "Container1");
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ClientSide/DynamicHeight/Container_Scroll_spec.js:10]: `entityExplorer.SelectEntityByName("Container1"); `is needed?

**Review by Aishwarya-U-R — APPROVED** [review]
Approving this now since its been long & there are still review comments to be taken care - which is assured to be taken care in next PR.
